client/packages/common/src/intl/locales/en/inventory.json

```diff
@@ -23,6 +23,8 @@
   "label.locked": "Locked",
   "label.batch": "Batch",
   "label.pricing": "Pricing",
+  "label.supplier": "Supplier",
+  "label.no-supplier": "Unknown",
```

Comment:
Could also be `N/A` or `Inventory Adjustment`... not to sure what to label stock that appear through stocktake 🤔